File: pgm/ABC_Java/ABC_354_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int H = Integer.parseInt (reader.readLine());
        int ans = 1;
        int base = 2;
        int e = 1;
        int h = 1;
        while (h <= H) {
            e *= base;
            h += e;
            ans++;
        }
        System.out.println (ans);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_354_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.util.StringTokenizer

// import kotlin.math.*

const val one: Int = 1
const val base: Int = 2

fun main() {
    val H: Int = readln().toInt()
    var day: Int = one
    var e: Int = one
    var value: Int = one
    while (value <= H) {
        e *= base
        value += e
        day += one
    }
    println(day)
    kotlin.system.exitProcess(0)
}